server/models.py

```diff
@@ -24,18 +27,66 @@ def process_result_value(self, value, engine):
         else:
             return set(value.split(','))
 
+
 class User(db.Model, UserMixin):
     __tablename__ = 'users'
     id = db.Column(db.Integer, primary_key=True)
-    email = db.Column(db.String(255), nullable=False, index=True)
-    offerings = db.Column(StringSet, nullable=False)
+    name = db.Column(db.String(255), index=True, nullable=False)
+    canvas_id = db.Column(db.String(255), nullable=False, index=True)
+    staff_offerings = db.Column(StringSet, nullable=False)
+    student_offerings = db.Column(StringSet, nullable=False)
+
+
+class Offering(db.Model):
+    __tablename__ = 'offerings'
```

Comment:
Yeah, this table really needs to be course_offerings

Reply:
```
class CourseOffering(db.Model)
		__tablename__ = 'course_offerings"
```

like this?